src/update_lmod_system_cache_files.in

```diff
@@ -334,7 +334,7 @@ set_defaults
 check_parameters
 
 if [ -z "$LMOD_REVERSEMAPT_DIR" ]; then
-  LMOD_REVERSEMAPT_DIR="$LMOD_CACHE_DIR/../reverseMapD"
+  LMOD_REVERSEMAPT_DIR="$LMOD_CACHE_DIR/reverseMapD"
```

Comment:
People may be relying on the current location, so changing this may affect them.

I'm not sure why it is the way it is, but this script probably mimics how it was before (not sure, I'd have to go back in time and check).

Why do you need to put it in two places if you want to get control?